workspaces/bulk-import/plugins/bulk-import/src/components/Repositories/AddedRepositoryTableRow.tsx

```diff
@@ -44,14 +45,18 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const ImportStatus = ({ data }: { data: AddRepositoryData }) => {
+const ImportStatusComponent = ({ data }: { data: AddRepositoryData }) => {
```

Comment:
You just renamed it without a reason? Maybe this happen with a merge conflict. I think `ImportStatus` was good.